server/service/src/sync/program_schemas/hiv_care_program.json

```diff
@@ -215,7 +200,7 @@
           "type": "string"
         }
       },
-      "required": ["text", "created", "authorId", "authorName"],
+      "required": ["text"],
```

Comment:
no mention in the PR or issue of why the required fields might need to be dropped. Is there a reason?

Reply:
All these schema changes are just imported from the latest version in the program schema repo. (I just re-exported them).

This one in particular (and maybe the others) are from what @clemens-msupply  did in #1463, which hasn't been merged yet. It's possible the other changes are from PRs that also aren't merged yet.

I'll co-ordinate with @clemens-msupply in the morning to clarify and make sure we merge in the correct order (so as to not end up with compile errors like above)

Reply:
Okay, the other schema diffs are gone now after a fresh merge from feature branch. Only this one remains and it's from #1463. So I won't merge this one until after that.